index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Building with Backgrounds and Gradients</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="#"><img src="images/apple-logo.svg" alt="apple logo"></a>
+        <li><a href="#">Store</a>
+        <li><a href="#">Mac</a>
+        <li><a href="#">iPod</a>
+        <li><a href="#">iPhone</a>
+        <li><a href="#">iPad</a>
+        <li><a href="#">iTunes</a>
+        <li><a href="#">Support</a>
+        <li>
+          <input type="text" placeholder="🔍">
```

Comment:
Hmmm 🤔 maybe there is a better way of adding the search icon, the placeholder is not best way.
Maybe you can address this in the future and create an issue out of it!

Reply:
Add `autofocus` attribute